ORM.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+import sqlalchemy as sql
+
+Base = declarative_base()
+
+
+def opendb():
+    engine = sql.create_engine('sqlite:///studentsoncourse.db')
+    Base.metadata.create_all(engine)
+    Session = sessionmaker(bind=engine)
+    return Session()
+
+
+class Students(Base):
+    __tablename__ = 'students'
+
+    id = sql.Column(sql.Integer, primary_key=True)
+    name = sql.Column(sql.String)
+    surname = sql.Column(sql.String)
+
+
+# Help lines
```

Comment:
This shouldn't be in the codebase.
Keep it in tests' fixtures or separate helper scripts